shell/platform/embedder/embedder_external_view.cc

```diff
@@ -114,10 +114,10 @@ bool EmbedderExternalView::Render(const EmbedderRenderTarget& render_target,
 
     impeller::TextFrameDispatcher collector(aiks_context->GetContentContext(),
                                             impeller::Matrix());
-    display_list->Dispatch(collector, sk_cull_rect);
 
     impeller::ExperimentalDlDispatcher impeller_dispatcher(
-        aiks_context->GetContentContext(), *impeller_target, cull_rect);
+        aiks_context->GetContentContext(), *impeller_target,
+        /*supports_readback=*/false, cull_rect);
```

Comment:
I didn't see a submit info in this class...

Reply:
I should be providing that info very soon now in DL and saveLayer...